### DISCUSSION TITLE
请问火狐版本如何安装？ 我目前下载了Firefox.zip，也改了火狐设置，之后要如何做？

### DISCUSSION THREAD

**Discussion by echosori** [crimx/ext-saladict#2069] [Q&A]
如题，谢谢！

**Comment by konzonz** [comment]
（确定已经改为忽略火狐插件认证，否则会阻止你安装）(适用Firefox ESR版)
1，打开扩展管理页面；

  _2，直接拖拽Firefox.zip到页面；_
或者：
  _2，在页面中找到齿轮按钮点击出子菜单；
  3，在菜单中选择从文件安装附加组件；
  4，在弹出的对话框中选择Firefox.zip；_
